consensus/src/chained_bft/test_utils/mod.rs

```diff
@@ -61,20 +61,6 @@ pub fn build_simple_tree() -> (
     (vec![genesis_block, a1, a2, a3, b1, b2, c1], block_store)
 }
 
-pub fn build_chain() -> Vec<Arc<ExecutedBlock<TestPayload>>> {
```

Comment:
Dead code?   didn't see a direct call to it anywhere else in the edits so assuming it was just in the way?